app/lib/features/spaces/widgets/public_spaces_selector.dart

```diff
@@ -464,11 +462,11 @@ class PublicSpaceSelector extends ConsumerWidget {
         ),
       ),
     ];
-    if (val != null && val.isNotEmpty) {
+    if (val?.isNotEmpty == true) {
```

Comment:
Do we really required` == true` here? 
I think `val?.isNotEmpty` should be sufficient.

Reply:
If ` == true` doesn't exist, lint error occurred.